docs/backups.md

```diff
@@ -195,9 +195,20 @@ The Operator will also need your service account key to access storage.
     type. These actions will result in downloading a file in JSON format with
     your new private key and related information.
 
-3. Now you should use a base64-encoded version of this file and create the [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/). You can encode
-    the file with the `base64 <filename>` command. When done, create a yaml file
-    with your cluster name and base64-encoded file contents as the following
+3. Now you should create the [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
+    using base64-encoded versions of two files: the file containing the
+    private key have just downloaded, and the special `gcs.conf` configuration file.
+
+    This `gcs.conf` file should have the following contents:
+
+    ```
+    [global]
+    repo1-gcs-key=/etc/pgbackrest/conf.d/gcs-key.json
```

Comment:
User can have 4 repos - repo1, repo2, repo3, repo4.
For repo2 it will be `repo2-gcs-key`. 
We should highlight it, that the content of the conf file depends on the repo.